src/components/underground.html

```diff
@@ -148,11 +148,11 @@ <h4><u>Treasures</u></h4>
 
                     <h4><u>Upgrades</u></h4>
                     <p>You can use your diamonds to buy all kinds of upgrades that will help you explore the underground even further!</p>
-                    <p>Some upgrades will make the Underground harder and are less cost efficient.
+                    <p>Some upgrades will make the Underground harder and are less cost-efficient.
                     <br/>Avoid these until you have the other available upgrades!</p>
 
                     <h4><u>Daily deals</u></h4>
-                    <p>In the Daily deals tab you can trade your items.</p>
+                    <p>In the Daily deals tab you can trade your items. The trades available rotate each day at midnight.</p>
```

Comment:
```suggestion
                    <p>In the Daily deals tab you can trade your items. The trades available change each day at midnight.</p>
```
`rotate`, to me, implies there is some fixed rotation where today's deals will eventually roll back around, but that is not the case.